spec.html

```diff
@@ -3633,26 +3633,18 @@ <h1>Await Rejected Functions</h1>
       </emu-clause>
 
       <emu-clause id="sec-normalcompletion" aoid="NormalCompletion">
-        <h1>NormalCompletion</h1>
-        <p>The abstract operation NormalCompletion with a single _argument_, such as:</p>
+        <h1>NormalCompletion ( _value_ )</h1>
+        <p>The abstract operation NormalCompletion takes argument _value_ (an ECMAScript language value or ~empty~). It performs the following steps when called:</p>
```

Comment:
```suggestion
        <p>The abstract operation NormalCompletion takes argument _value_. It performs the following steps when called:</p>
```

Given that the type isn't accurate, best to leave it out for now, I think (even though the inaccuracy exists elsewhere).

Reply:
Urgh, okay, done.

Reply:
![image](https://user-images.githubusercontent.com/218840/126190363-dc52bc74-28fc-41c3-9a3c-b482e18f243e.png)

How is it inaccurate?

Reply:
Normal completions hold values which are not ES language values or `~empty~`. The table you've screenshotted is wrong; that's the "elsewhere" I was referring to.

(For example, PropertyBindingInitialization returns a completion record holding a list. jmdyck has a larger list of examples [here](https://github.com/tc39/ecma262/issues/496#issuecomment-491603203).)

Reply:
For *explicit* calls to NormalCompletion, I suspect `an ECMAScript language value or ~empty~` actually is accurate. However, there are also [implicit calls](https://tc39.es/ecma262/#sec-implicit-completion-values) to NormalCompletion, and those get us into issue #496. I.e. the spec is inconsistent, and one possible resolution of that inconsistency is that the `[[Value]]` field of a completion record can be many more things than just `an ECMAScript language value or ~empty~`.